### ISSUE TITLE
Error: Can't resolve './font-awesome-styles' when upgrading to webpack@2.1.0-beta.6

### ISSUE THREAD

**Issue by jimbolla** [gowravshekar/font-awesome-webpack#20]
If I upgrade from webpack 1.x to 2.1.0-beta.6 I get the error

```
ERROR in ./~/font-awesome-webpack/index.js
Module not found: Error: Can't resolve './font-awesome-styles' in 'C:\Projects\git\AppCenter\AppCenter.WebApp\Assets\node_modules\font-awesome-webpack'
 @ ./~/font-awesome-webpack/index.js 1:0-72
```

The error goes away if I edit `...\node_modules\font-awesome-webpack\index.js` from

```
require("style!css!less!./font-awesome-styles!./font-awesome.config.js");
```

to

```
require("style!css!less!./font-awesome-styles.loader!./font-awesome.config.js");
```

**Comment by boyarskiy** [comment]
Yes, same issue for me.

**Comment by boyarskiy** [comment]
Same issue for me.

**Comment by JoeMattie** [comment]
Here's an entirely userspace fix (no need to edit the file in node_modules)

```
import 'style!css!less!font-awesome-webpack/font-awesome-styles.loader!font-awesome-webpack/font-awesome.config.js';
```

or

```
require('style!css!less!font-awesome-webpack/font-awesome-styles.loader!font-awesome-webpack/font-awesome.config.js');
```

**Comment by yilinglu** [comment]
Should this fix by JoeMattie be integrated into the index.js file in the source code?

**Comment by eMerzh** [comment]
Hi, 
still have the issue with webpack2 too.. is it possible to fix this before the release of the stable release?

**Comment by trsh** [comment]
The provided fix also not working

**Comment by kopax** [comment]
Still have the issue with webpack2 as well

**Comment by karvapallo** [comment]
Yeah I'm still having this issue as well. The fix given by @JoeMattie did not work for me.

**Comment by wolfadex** [comment]
Release 0.0.5-beta.2 should fix this issue for you, a release 0.0.6 still needs to be made so that this is no longer an issue with normal installs of font-awesome-webpack.

Sorry for the delay in getting 0.0.6 out, I'll try to get it out soon.
